apps/server/src/modules/provisioning/strategy/oidc/service/schulconnex-user-provisioning.service.ts

```diff
@@ -70,4 +75,18 @@ export class SchulconnexUserProvisioningService {
 
 		return savedUser;
 	}
+
+	private async checkUniqueEmail(
+		email: string,
+		systemId: string,
```

Comment:
You pass schoolId and systemId just for the loggable, right? 
In my opinion this isnt necessary. The information about the email address and externalId should be enough.